internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -535,6 +554,42 @@ if ($with_fluentd) {
     echo "- Started"
 }
 
+if ($with_dotnet_instrumentation) {
+    echo "Installing SignalFx Instrumentation for .NET ..."
+    $api = "https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases/latest"
+    $module_name = "Splunk.SignalFx.DotNet.psm1"
+    echo "Downloading .NET Instrumentation installer ..."
+    $download = (Invoke-WebRequest $api | ConvertFrom-Json).assets | Where-Object { $_.name -like $module_name } | Select-Object -Property browser_download_url,name
+    $dotnet_auto_path = Join-Path $tempdir $download.name
+    Invoke-WebRequest -Uri $download.browser_download_url -OutFile $dotnet_auto_path
+    Import-Module $dotnet_auto_path
+    echo "Installing SignalFx Dotnet Auto Instrumentation..."
+    Install-SignalFxDotnet
+
+    echo "Setting environment variables for instrumentation ..."
+    update_registry -path "$regkey" -name "COR_ENABLE_PROFILING" -value "1"
```

Comment:
Per meeting today let's exclude the COR* and CORECLR* env vars from here.

Reply:
I removed all of the env vars that were set here. Do some need to stay? If so, perhaps they could be set by the instrumentation install script?